packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -39,7 +42,21 @@ export interface FileDownloadCards {
 export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
   const truncateLength = 15;
   const { userId, message } = props;
+  // @conditional-compile-remove(file-sharing)
+  const [showSpinner, setSpinner] = useState(false);
   const fileDownloads: FileMetadata[] = message.metadata ? extractFileMetadata(message.metadata) : [];
+  const DownloadIconTrampoline = (): JSX.Element => {
+    // @conditional-compile-remove(file-sharing)
+    if (showSpinner) {
+      // @conditional-compile-remove(file-sharing)
+      return <Spinner size={SpinnerSize.medium} aria-live={'assertive'} />;
+    } else {
+      // @conditional-compile-remove(file-sharing)
+      return <ChatCompositeIcon iconName="Download" />;
+    }
+    // Return _some_ available icon, as the real icon is beta-only.
+    return <ChatCompositeIcon iconName="EditBoxCancel" />;
```

Comment:
No need. Nothing should be conditional in this function

Reply:
Done